### ISSUE TITLE
Why server pads tiddler text with the line feed character (\n) when saving them

### ISSUE THREAD

**Issue by zimiarh** [OokTech/TW5-MultiUser#20]
First of all, thanks again for making this plugin. I have been using it in my own project for a while and it does satisfy my needs. However, some glitches still might happen occasionally and make me bumping my head to the wall several times. 

One of the thing that bites my most frequently is that this plugin pads tiddler text with the line feed character and substantially making the tiddler text differs from what a user inputs. For most of the time, this behaviour harms no one as it only adds a newline to the tiddler. Not a big deal. However, sometimes we do not allow any single character of the tiddler text being changed. For example, some other plugins (and TiddlyWiki itself) saves configurations of their program in the text of **config** tiddlers. When a user (that is, me) changes the text of these tiddlers, the extra line feed character breaks things up unexpectedly.

Although I could have quietly changed such behaviour in my own codebase, I am curious about what is the reason to wrap the texts with line feed character. My guess is that *nix tools requires text files having a trailing line feed character to work properly. Is this the main reason? If so, will it be safe to stop padding line feed character as TiddlyWiki seems not depending on *nix tools.

By the way, currently this plugin adds a pair of endline character to the beginning and the end of the text **whenever a tiddler is being saved**. Which means that if I save a tiddler three times, there will be three line feeds added to the text.

**Comment by inmysocks** [comment]
Are you sure that you are using the newest version? I had that problem but now all the files have whitespace trimmed before they are saved.

**Comment by zimiarh** [comment]
My branch basically adds the ability to queue modifications of tiddlers on the client side and allows a user to manually submit those modifications to server. The branch is non-mergeble because I chose to remove waiting list on the server side to avoid problems when modifications are submitted manually. However, I found that is a bad decision so anyone should better not invest more time on that direction. I might re-write this feature base on the latest master branch if I have time, although I don't know when that is going to happen.

Thanks for your help on my issues! Hope this plugin can get better and better!
